snuba/datasets/events_format.py

```diff
@@ -72,13 +87,19 @@ def extract_extra_contexts(contexts) -> Tuple[Sequence[str], Sequence[str]]:
                     value = _unicodify(ctx_value)
                     if value:
                         ctx_key = f"{ctx_name}.{inner_ctx_name}"
-                        context_keys.append(_unicodify(ctx_key))
-                        context_values.append(_unicodify(ctx_value))
+                        unicode_key = _unicodify(ctx_key)
+                        unicode_value = _unicodify(ctx_value)
+
+                        assert isinstance(unicode_key, str)
+                        assert isinstance(unicode_value, str)
```

Comment:
What about these two? Are we sure they are non nullable strings. and if yes can we change the signature of the method ?

Reply:
Yes. Similarly to tags, the key is definitely a string because of `as_dict_safe` and the value is definitely a string because it's checked on line 88 above.

Reply:
ok